components/forms/w-textfield.vue

```diff
@@ -51,6 +51,7 @@ const inputClasses = computed(() => ({
         ]"
         :autocomplete="autocomplete"
         :disabled="disabled"
+        :placeholder="placeholder"
```

Comment:
What I don't understand is why we didn't need this in Fabric ([here's f-input component code](https://github.com/fabric-ds/vue/blob/main/components/forms/f-input.vue#L6)). It felt like `...$attrs` did the job there.

Reply:
I think I understand now. Inside `<script>` we need to use `p.placeholder` value to determine whether a respective `ccInput.placeholder` class should be applied. So we separated out `placeholder` prop from the attributes. Then we no longer have access to that attribute through ...$attrs inside the `<template>`. In Fabric we didn't need to define `placeholder` prop separately in order to apply styling, so the attribute was passed together with other attributes to the input element. At least that's how I see it :D

Reply:
Yup. Any attributes not specified as props will go in to attrs and that way get added to the input field. So when placeholder was added as a prop it would have needed to be added here as well.